Testing/Regression/PeleLM-tests.ini

```diff
@@ -400,3 +400,29 @@ useOMP = 1
 numthreads = 2
 compileTest = 0
 doVis = 0
+
+[EB_WaveSpecPastCyl2D_AMR]
+buildDir = Exec/RegTests/EB_Convection_Wave_Species
+inputFile = inputs.2d-regt
+probinFile = probin.2d.regt
+dim = 2
+restartTest = 0
+useMPI = 1
+numprocs = 4
+useOMP = 1
+numthreads = 2
+compileTest = 0
+doVis = 0
+
+[EB_FS2D]
+buildDir = Exec/RegTests/EB_FlameSheet
+inputFile = inputs.2d-regt
+probinFile = probin.3d.test
+dim = 2
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 2
+compileTest = 0
+doVis = 0
```

Comment:
sorry, can you use the newer abbreviated test naming scheme so that it fits in the table like the others?

Reply:
Also, the test are listed in the wrong folder